uint/benches/bigint.rs

```diff
@@ -637,8 +637,8 @@ fn from_fixed_array(c: &mut Criterion) {
 		[255, 0, 0, 123, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 121, 0, 0, 0, 0, 0, 213, 0, 0, 0, 0, 0, 0];
 	c.bench_function("from_fixed_array", move |b| {
 		b.iter(|| {
-			let _: U512 = black_box(ary512.into());
-			let _: U256 = black_box(ary256.into());
+			let _: U512 = black_box(black_box(ary512).into());
+			let _: U256 = black_box(black_box(ary256).into());
```

Comment:
This has a very significant impact on the benchmark, up to 5x difference on my machine. Good catch.

I find that it difficult to build a good intuition for when the blackbox is necessary; I wish there was better tooling around that.

Reply:
So far a good strategy seems to be to put every input param and the final result in the black_box and remove some if there is no difference.

Reply:
Not sure if that qualifies as a "good" strategy, but yeah, that sounds like the best we can do. :/